example_implementation/scaler.py

```
@@ -13,7 +13,7 @@
     logging.getLogger(PACKAGE)
 
     scaler = Scaler(
-        redis_conn=Redis(decode_responses=True), stream=STREAM, consumer_group=GROUP
+        redis_conn=Redis(decode_responses=True,ssl=True,ssl_cert_reqs=None, username="default",password="Acj-AAIncDE3NDYwYWEzNTUwNjk0YmI3ODAxM2VhMDc2MTA4NzczM3AxNTE0NTQ", host="honest-hedgehog-51454.upstash.io"), stream=STREAM, consumer_group=GROUP
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "password"

(S106)